setuptools.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: setuptools
+  version: 69.2.0
+  epoch: 1
+  description: Easily download, build, install, upgrade, and uninstall Python packages
+  copyright:
+    - license: "MIT"
+  dependencies:
+    provides:
+      - whl:setuptools=${{package.version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-pip
+      - py3-wheel
+      - python-3.10-base
+      - python-3.11-base
+      - python-3.12-base
```

Comment:
i was thinking to add a 'pythons-meta.yaml' package (or similar name) with a 'pythons-base' and 'pythons-base-dev'.